src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2476,6 +2476,74 @@ def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certifi
                                                 certificate_envelope=kv_cert_def)
 
 
+def sync_kv_cert(cmd, resource_group_name, name, key_vault, thumbprint=None):
+    Certificate = cmd.get_models('Certificate')
+    client = web_client_factory(cmd.cli_ctx)
+    webapp = client.web_apps.get(resource_group_name, name)
+    if not webapp:
+        err_msg = "'{}' app doesn't exist in resource group {}".format(name, resource_group_name)
+        raise ResourceNotFoundError(err_msg)
+    server_farm_id = webapp.server_farm_id
+    location = webapp.location
+
+    key_vault_id = None
+    if not is_valid_resource_id(key_vault):
+        kv_client = get_mgmt_service_client(cmd.cli_ctx, ResourceType.MGMT_KEYVAULT)
+        key_vaults = kv_client.vaults.list_by_subscription()
+        for kv in key_vaults:
+            if key_vault.lower() == kv.name.lower():
+                key_vault_id = kv.id
+                break
+    else:
+        key_vault_id = key_vault
+
+    if thumbprint:
+        certs = list(client.certificates.list_by_resource_group(resource_group_name,
+                                                                filter=f"ServerFarmId eq {'server_farm_id'}"))
```

Comment:
I think the single quotes should be outside of the `{}`. Also, just double checking but `filter` won't be used until the SDK/swagger is updated right?